chessington/engine/pieces.py

```diff
@@ -49,7 +83,36 @@ class Knight(Piece):
     """
 
     def get_available_moves(self, board):
-        return []
+        available_moves = []
+        current_square = board.find_piece(self)
+        possible_squares = self.get_possible_squares(board, current_square)
+
+        for square in possible_squares:
+            if board.get_piece(square) is None:
+                available_moves.append(square)
+            elif board.get_piece(square).player != self.player:
+                available_moves.append(square)
```

Comment:
Instead of defining another variable `available_moves` and append the correct moves, can't we remove the `unavailable_moves` from the list?